stories/file-uploader/file-uploader.stories.js

```diff
@@ -23,32 +23,30 @@ Use the File Uploader if:
 
 export const primary = () => `
 <div class="fd-form-item">
-    <div class="fd-file-uploader">
-        <input 
-          class="fd-input fd-file-uploader__input" 
-          onclick="browseFile('input1');" 
-          title="Select a file for uploading" 
-          aria-label="Select a file for uploading"
-          aria-live="polite"
-          autocomplete="off"
-          type="text" 
-          id="browse_input1" 
-          placeholder=" Choose a file for upload" 
-          tabindex="-1"
-          readonly>
-        <button 
-          class="fd-button"
-          onclick="browseFile('input1');" 
-          tabindex="0"
-          id="file-uploader-button-1" 
-          aria-label="Select a file for uploading">Browse...
-        </button>
-    </div>
-    <input
-      id="input1"
-      class="fd-file-uploader__hidden"
-      type="file"
-      onchange="selectFile(this,'browse_input1')">
+  <div class="fd-file-uploader">
+    <label for="browse_input1" class="fd-file-uploader__hidden">File upload</label>
+    <input 
+      class="fd-input fd-file-uploader__input" 
+      onclick="browseFile('input1');" 
+      title="Choose a file for upload" 
+      type="text" 
+      id="browse_input1" 
+      autocomplete="off"
```

Comment:
I'm not entirely sure it's necessary here since this input should only ever be `readonly`, but it was already here so I wasn't sure if there was a reason. I don't think even Chrome would add predictions to a readonly field.